source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js

```diff
@@ -56,6 +56,7 @@
         }
         this.save = function () {
             var updateCalificators = this.calificatorsManagerModel.toJs();
+            var updateEvaluators = this.calificatorsManagerModel.toEvaluators();
```

Comment:
This method update the array of the evaluators and just returns the names, because it's the only thing that we need... Do you sugest to change the name maybe to 'evaluatorsName' or 'toEvaluatorsName'